README.md

```diff
@@ -3,18 +3,23 @@
 The IBM block storage CSI driver enables container orchestrators, such as Kubernetes and OpenShift, to manage the life-cycle of persistent storage.
 
 Supported container platforms:
-  - OpenShift v4.2
-  - Kubernetes v1.14
+  - Kubernetes v1.14 (x86 architecture)
+  - Kubernetes v1.16 (x86 architecture)
+  - OpenShift v4.2 (x86, IBM Z architectureS)
+  - OpenShift v4.3 (x86 architecture)
 
 Supported IBM storage systems:
   - IBM FlashSystem 9100
+  - IBM FlashSystem A9000/R
   - IBM Spectrum Virtualize
   - IBM Storwize
-  - IBM FlashSystem A9000/R
-
+  - IBM DS8880
+  - IBM DS8900
+  
 Supported operating systems:
-  - RHEL 7.x (x86 architecture)
-
+  - Red Hat Enterprise Linux 7.x (x86 architecture)
+  - Red Hat Enterprise Linux CoreOS (x86, IBM Z architectureS)
```

Comment:
`architechtureS`

Reply:
make this lower case "s"

Reply:
this already fixed

Reply:
the "S" case wasn't lowered as rivka suggested, it was removed:
https://github.com/IBM/ibm-block-csi-operator/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21